### ISSUE TITLE
Remember window size and position

### ISSUE THREAD

**Issue by mattwltr** [melmanm/UltraWideScreenShare-2#8]
It would be great if UltraWideScreenShare-2 would remember and restore the last size and position on screen after it was closed and reopended.

**Comment by ertan2002** [comment]
Yes it would be great. As it is c#, I can make it probably. I'll download this repo and work on it later.

**Comment by melmanm** [comment]
In case the application is installed from [Microsoft Store](https://apps.microsoft.com/detail/9p6j8n3k7tk4) Windows takes care of restoring application with the size and position it was closed.

If you would like to use standalone version I have prepared new release [Release v1.0.0-issue8](https://github.com/melmanm/UltraWideScreenShare-2/releases/tag/v1.0.0-issue8) which has this functionality implemented. 

So far I will not merge #9, since it duplicates Windows functionality.
